tests/tasks/airbyte/test_airbyte.py

```diff
@@ -251,3 +248,23 @@ def test_get_job_status_2(self):
         job_id = "1"
         with pytest.raises(JobNotFoundException):
             task._get_job_status(session, airbyte_base_url, job_id)
+
+    # airbyte export tests
+    def test_construction(self):
+        task = AirbyteConfigurationExport()
+        assert task.airbyte_server_host == "localhost"
+        assert task.airbyte_server_port == 8000
+
+    @responses.activate
+    def test_export_configuration(self):
+        airbyte_base_url = "http://localhost:8000/api/v1"
+        responses.add(
+            responses.POST,
+            airbyte_base_url + "/deployment/export/",
+            body="\x02\x03\x05\x07",
+            status=200,
+        )
+        task = AirbyteConfigurationExport()
+        session = requests.Session()
+        airbyte_config = task._export_configuration(session, airbyte_base_url)
```

Comment:
We want to make sure we're testing the run method since that's the user facing part of the task.
```suggestion
        airbyte_config = task.run()
```

Reply:
updated